snapshot/snapshot.go

```diff
@@ -203,11 +203,17 @@ func NewSnapshotter(ctx context.Context, cfg *config.Config) (snapshots.Snapshot
 		return nil, err
 	}
 
+	syncRemove := cfg.SyncRemove
+	if cfg.FsDriver == config.FsDriverFscache {
+		log.L.Infof("for fscache mode enable syncRemove")
```

Comment:
We can not use syncRemove flag to control whether to remove fscache files. Just directly remove it in a goroutine

Reply:
AKAIK, the syncRemove is used for making removing snapshot in order , cause we need remove bootstrap snapshot first and then we can umount erofs. So should we force enable syncRemove for fscache ?

Reply:
We can simply make fscache files deletion out of the syncRemove==true if arm